core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -72,4 +77,85 @@ class SparkContextSuite extends FunSuite with LocalSparkContext {
     val byteArray2 = converter.convert(bytesWritable)
     assert(byteArray2.length === 0)
   }
+
+  test("addFile works") {
+    val file = new File("somefile")
```

Comment:
I think we should use one of the `createTempFile` methods to create this file, rather than using the CWD.